game/src/ltn/route.rs

```diff
@@ -60,6 +71,9 @@ impl RoutePlanner {
         .ignore_initial_events()
         .build(ctx);
 
+        new.restore(ctx, &self.panel);
```

Comment:
Note this isn't keeping the spinner value currently because `Spinner` doesn't actually implement `can_restore` in widgetry. :(  I think we could add it, but I should probably audit all the places using spinners now to make sure they don't break